.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
```

Comment:
I just wanted to point out that this will result in "[Bug]: " being pre-filled in the issue title field. However, we do not really use "[Bug]:" or "[Feature]:" prefixes in issue titles in the ROS 2 repos.

@audrow thoughts?

Reply:
Good point. I suppose it should be removed. We generally seem to prefer labels.

Reply:
Thanks, I removed both.